editor/editor_feature_profile.cpp

```diff
@@ -64,7 +64,10 @@ void EditorFeatureProfile::set_disable_class(const StringName &p_class, bool p_d
 }
 
 bool EditorFeatureProfile::is_class_disabled(const StringName &p_class) const {
-	return disabled_classes.has(p_class);
+	if(!ClassDB::class_exists(p_class)) {
```

Comment:
There should be a space between `if` and `(`, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html